src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -65,19 +66,28 @@ protected function doSend(MessageInterface $message): SentMessage
         $post = [
             '$type' => 'app.bsky.feed.post',
             'text' => $message->getSubject(),
-            'createdAt' => (new \DateTimeImmutable())->format('Y-m-d\\TH:i:s.u\\Z'),
+            'createdAt' => \DateTimeImmutable::createFromFormat('U', time())->format('Y-m-d\\TH:i:s.u\\Z'),
         ];
         if ([] !== $facets = $this->parseFacets($post['text'])) {
             $post['facets'] = $facets;
         }
 
+        $options = $message->getOptions()?->toArray() ?? [];
+        $options['repo'] = $this->authSession['did'] ?? null;
+        $options['collection'] = 'app.bsky.feed.post';
+        $options['record'] = $post;
+
+        if (isset($options['attach'])) {
```

Comment:
In the if we need to check if Symfony/mime is installed.

You can search the codebase for "Try running..." to find an example

Reply:
Not sure.

The `BlueskyOptions::attachMedia` cannot be called if the component isn't installed because it has the `Symfony\Component\Mime\Part\File` in the function parameter.

Reply:
Interesting, any advice @nicolas-grekas ? Shall we make it required then?

We can also check it in the constructor of the options class, but as new options (setter) can come, it is not sure if we need the package in the end or not.

Reply:
Another option would be to go with the file path and construct the file object later where we can then check at runtime for `symfony/mime`

Reply:
I like your proposal.

But it's not consistent with existing code.

I don't know if it's a problem or not ?

Reply:
Which code do you mean for comparison? It's not part of this notifier package right?

Reply:
This is the code use for the Mastodon bridge for example:

https://github.com/symfony/symfony/blob/eccdbea19f78a1c140f0f838ae7165717b855e14/src/Symfony/Component/Notifier/Bridge/Mastodon/MastodonOptions.php#L52-L54

Reply:
Ok, lets do it [this way](https://github.com/symfony/symfony/pull/54843)

I fixed the Mastodon bridge.

Thanks

Reply:
Ok I closed my PR, let's just move it to require-dev and remove the check. Sorry for the back and forth